core/sr-arithmetic/benches/bench.rs

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 Parity Technologies
```

Comment:
wrong license - please ensure it's the same text as the file from which the code has been copied.